pre-requisite-installations.asciidoc

```diff
@@ -365,7 +368,9 @@ activate script that was compatible with Git-Bash. Reinstall the latest Python
 
 *******************************************************************************
 
-
+// JAN: I'd suggest to use exact Selenium version as well for two reasons
+// 1. To avoid readers issues because of breaking changes in Selenium updates. I had a problem with poetry where installing the latest versions instead of specific one broke the whole course.
+// 2. To not raise eyebrows - why is there version for Django but not for selenium
```

Comment:
thanks for this too! 

So the problem with pinning selenium, in my experience from previous editions, is that selenium is in a race to keep up with browsers.  So newer versions of firefox sometimes need a newer selenium, and we cant ask readers to pin firefox
So instead what i do is try and always use the latest -- In CI, the book tests install selenium fresh every time, latest version.  (and django too actually but that's a separate issue)

but you're right, the discrepancy needs explaining.  i'll add a note.